RNA-seq/01-qc_trim_quant.md

```diff
@@ -0,0 +1,244 @@
+## Introduction to bulk RNA-seq data processing
+
+We will first learn how to process RNA-seq data at the command line using two samples that were assayed with paired-end sequencing.
+
+These samples come from a project ([`PRJNA178120`](https://www.ebi.ac.uk/ena/data/view/PRJNA178120)) that includes 8 samples from normal gastric tissue, gastric cancer cell lines and primary gastric tumor cell cultures.
+
+Here we will perform quality control checks, trimming, and estimate the transcript abundances for a single sample, SRR585570.
+
+**Our objectives:**
+
+* Get comfortable with the command line
+* Learn about project organization
+* Become familiar with [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/), preprocessing with [fastp](https://github.com/OpenGene/fastp) and quantification with [Salmon](https://combine-lab.github.io/salmon/)
+
+Later, we will use the full dataset (n = 8) to explore how to summarize estimates to the gene level and do some exploratory data analyses with data the course directors have processed ahead of time.
+
+---
+
+For these exercises, we'll want to set our **LOCAL FOLDER** in Kitematic to the `RNA-seq` folder.
+
+Copy and paste the text in the code blocks below into your `Terminal` window in RStudio.
+It should be in the lower left hand corner as a tab next to `Console`.
+
+We'll first want to set our working directory to the top-level of the RNA-seq folder like so
+
+```bash
+cd kitematic
+```
+
+**Because these steps are computationally time intensive, we've prepared a script to start running things.**
+Once we start running the script, we will give a short lecture to introduce this module and then walk through and explain each of the individual steps that the script is executing.
+
+**Enter the following in the Terminal to start running the script:**
+
+```bash
+./script/run_SRR585570.sh
+```
+
+_Note: Don't worry if the Salmon step does not complete by the time we move on to the next notebook.
+This is a time and resource intensive step, so we have prepared the required output in case we need it._
+
+### Input files
+
+The raw data FASTQ files (`fastq.gz`) for this sample, SRR585570, are in `data/fastq/gastric_cancer/SRR585570`.
+The first two directories, `data/fastq`, tell us that these files are data and 
+The third directory `gastric_cancer` tells us which experiment or dataset these data are from. 
```

Comment:
```suggestion
the third directory `gastric_cancer` tells us which experiment or dataset these data are from. 
```

Reply:
I think these sentences might have gotten lost/jumbled in your edits. Can you re-work them or you could also probably remove some of this entirely. Another thought is thatt this may be stuff you explain in person, but it feels a little obvious that the `data` folder would be for data. So I'm not sure if your original intention for this got lost or what this is intending to teach me here.

Reply:
I'm reworking them. Note one of the stated objectives is to learn about project organization and that's not really covered in the introduction to R that comes before this. It is purposely obvious here (I will probably say as much in person) and then how we talk about it becomes less obvious throughout this document. 

I'll also note that having all files in the same directory is something I've commonly seen folks with less experience do.